src/components/landing/Login.js

```diff
@@ -51,35 +54,43 @@ class Login extends Component {
     }
 
     return (
-      <StyledArea>
-        <StyledWrapper>
-          <StyledGreeting>Welcome!</StyledGreeting>
-          <StyledCaption>Let's get things done today.</StyledCaption>
-          <StyledForm onSubmit={this.handleSubmit}>
-            <StyledInput
-              type="text"
-              placeholder="Email"
-              value={this.state.email}
-              onChange={this.handleChangeEmail}
-              pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
-              maxLength="255"
-              title="Must provide a valid email"
-              required />
-
-            <StyledInput
-              type="password"
-              placeholder="Password"
-              value={this.state.password}
-              onChange={this.handleChangePassword}
-              pattern=".{6,}"
-              title="Must contain at least 6 or more characters"
-              required />
-            {errorMessage}
-            <StyledSubmit type="submit" value="Login" />
-
-          </StyledForm>
-        </StyledWrapper >
-      </StyledArea>
+      <StyledBody>
+        <StyledLandingBackground>
+          <StyledPopupContainer>
+            <StyledGreeting>Welcome!</StyledGreeting>
+            <StyledCaption>Let&apos;s get things done today.</StyledCaption>
+            <StyledForm onSubmit={this.handleSubmit}>
+              <StyledInput
+                type="text"
+                placeholder="Email"
+                value={this.state.email}
+                onChange={this.handleChangeEmail}
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
+                maxLength="255"
+                title="Must provide a valid email"
+                required />
+
+              <StyledInput
+                type="password"
+                placeholder="Password"
+                value={this.state.password}
+                onChange={this.handleChangePassword}
+                pattern=".{6,}"
+                title="Must contain at least 6 or more characters"
+                required />
+              {errorMessage}
+              <StyledSubmit type="submit" value="Login" />
+
+            </StyledForm>
+          </StyledPopupContainer >
+        </StyledLandingBackground>
+        <StyledAnimatedStacks>
```

Comment:
This can probably be moved it its own component as well, carrying the styles with it so Landing.js isn't bloated with style